com.spacestation14.Launcher.yaml

```diff
@@ -0,0 +1,37 @@
+---
+app-id: com.spacestation14.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+build-options:
+  append-path: "/usr/lib/sdk/dotnet6/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+  env:
+    PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+
+command: /app/ss14/bin/SS14.Launcher
+
+finish-args:
+  - --device=dri
+  # Allow all-device access for MIDI device support
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  # TODO: Replace this with fallback-x11 once Wayland support is available.
+  #       <https://github.com/AvaloniaUI/Avalonia/pull/8003>
+  #       <https://github.com/space-wizards/RobustToolbox/issues/2468>
+  - --socket=x11
+  # Allow access to the Music directory for MIDI files
+  - --filesystem=xdg-music
+  - --env=DOTNET_ROOT=/app/lib/dotnet
+
+modules:
+  - modules/fluidsynth.yaml
+  - modules/space-station-14-launcher.yaml
+  - modules/space-station-14-loader.yaml
+  - modules/extra-data.yaml
+  - modules/dotnet.yaml
```

Comment:
there is little reason to split these in separate manifest. it's harder to review maintain, etc.

(except those auto generated)